Update back-end-server.md (#2137)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Skype/SfbServer/plan-your-deployment/high-availability-and-disaster-recovery/back-end-server.md

  
## AlwaysOn Availability Groups and AlwaysOn Failover Cluster Instances

Skype for Business Server supports AlwaysOn Availability Groups only as active/passive, not active/active. 
  
To use AlwaysOn Availability Groups or AlwaysOn Failover Cluster Instances, you first use SQL Server to set up and configure the high availability solution. You can then use Topology Builder to associate it with a Front End pool.

Skype for Business Server supports AlwaysOn with the following database software:

- SQL Server 2017 Enterprise Edition

- SQL Server 2017 Standard Edition with limitations, see note below

- SQL Server 2016 Enterprise Edition

- SQL Server 2016 Standard Edition with limitations, see note below

- SQL Server 2014 Enterprise Edition
    
- SQL Server 2012 SP2 and CU2 Enterprise Edition

> [!NOTE]
> SQL Server 2017 and SQL Server 2016 are the only versions supported by Skype for Business Server 2019.

> 

Reviewer: ```suggestion
Always On Availability Groups is **not** supported in SQL 2016 and 2017 Standard Editions but you can use Always On Failover Cluster Instances. See [Editions and supported features of SQL Server 2016](https://docs.microsoft.com/sql/sql-server/editions-and-components-of-sql-server-2016?view=sql-server-2017) to learn more.
```

> > [!NOTE]

Reviewer: ```suggestion

```

> > SQL 2016 and 2017 Standard Edition limitation: Always On availability groups is not supported, you can only choose Always On failover cluster instances.

Reviewer: ```suggestion
> Always On Availability Groups is **not** supported in SQL 2016 and 2017 Standard Editions but you can use Always On Failover Cluster Instances. See [Editions and supported features of SQL Server 2016](https://docs.microsoft.com/sql/sql-server/editions-and-components-of-sql-server-2016?view=sql-server-2017) to learn more.
```

> > Refer to below link:

Reviewer: ```suggestion

```

> > https://docs.microsoft.com/en-us/sql/sql-server/editions-and-components-of-sql-server-2016?view=sql-server-2017

Reviewer: ```suggestion

```
